### DISCUSSION TITLE
[Feature] Logs view in /admin/system-settings

### DISCUSSION THREAD

**Discussion by t18n** [immich-app/immich#3484] [Feature Request]
### The feature

Would be awesome to be able to view system wide Logs in the Admin page on Web, so we can monitor what is currently happpening, what files are synced and what are the problems.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by gschintgen** [comment]
+1
Having readily accessible logs is quite helpful in other self-hosted apps.

**Comment by brent20** [comment]
+1 this would be an excellent addition and really help with quick troubleshooting. Thanks!
